web/annotation_editor_layer_builder.css

```
@@ -26,6 +26,15 @@
   /*#endif*/
 }
 
+@media (min-resolution: 144dpi) {
```

Looking at how we handled hiDPI images in the previous viewer UI, should we perhaps use something a little more generic here?
```suggestion
@media (min-resolution: 1.1dppx) {
```